### ISSUE TITLE
Ebay Partner Network Affiliate Link

### ISSUE THREAD

**Issue by wittyapps** [pajaydev/ebay-node-api#65]
Hello, are we able to add the affiliate data to the finding api input fields. see example below.

ebay.findItemsByKeywords({
            keywords: keywords,
            pageNumber: 1,
            limit: 30,
            affiliate: {
                customId: "search-ebay",
                networkId: "9",
                trackingId: "123456"
            }
        }).then((data) => {

});

**Comment by pajaydev** [comment]
@wittyapps  I will check and let you know

**Comment by pajaydev** [comment]
@wittyapps  Fixed it in `ebay-node-api@2.7.5`
Kindly use like below

```javascript
ebay.findItemsByKeywords({
    keywords: 'iphone',
    sortOrder: 'PricePlusShippingLowest', //https://developer.ebay.com/devzone/finding/callref/extra/fndcmpltditms.rqst.srtordr.html
    Condition: 3000,
    SoldItemsOnly: false,
    affiliate: {
        networkId: 9,
        trackingId: 1234567890
    }
})```
